src/modules/test/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ endif()
 
 gtest_discover_tests(modulestest
     EXTRA_ARGS ${CMAKE_CURRENT_BINARY_DIR}/testplate.json
+    XML_OUTPUT_DIR ${GTEST_OUTPUT_DIR}

```

Comment:
why isn't the `TestRecipes` test suite not getting picked up in CI. We clearly see the tests being run?

Reply:
I looked at the raw output of those tests and they do not seem to be registered correctly with gtest. For example:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites tests="0" failures="0" disabled="0" errors="0" time="0" timestamp="2022-06-07T20:16:24" name="AllTests">
</testsuites>
```

is missing the `<testsuite>...</testsuite>` information and `tests="0"`.

Reply:
Once the tests are registered there and outputting the complete XML results, they will automatically show in the reports.

Reply:
Let's address this issue when officially integrating the functional tests into our CI.